### PR TITLE
Adding blobber role

### PR DESCRIPTION
**Pull request by parithosh**
https://github.com/marioevz/blobber

### CODE REVIEW

File: roles/ethereum_node/tasks/validations.yaml [hunk 1]
```diff
@@ -13,3 +13,10 @@
     success_msg: "{{ ethereum_node_cl }} is a supported client"
     fail_msg: >
       ethereum_node_el = '{{ ethereum_node_cl }}' is invalid. You can choose between: '{{ ethereum_node_supported_cl }}'
+
+- name: Check if consensus client supports blobber configuration
+  ansible.builtin.assert:
+    that:
+      - ethereum_node_blobber_enabled != true or (ethereum_node_cl != 'nimbus' and ethereum_node_cl != 'teku')
```
Comment (skylenet) [inline roles/ethereum_node/tasks/validations.yaml:20]: Not sure if it's just me, but my brain got wonky reading this condition. Maybe we could use the `ansible.builtin.fail` here and do something like:

```yaml
ansible.builtin.fail:
  msg: "Error: The blobber tool currently doesn't support nimbus or teku" 
when: ethereum_node_blobber_enabled and ( (ethereum_node_cl != 'nimbus' or ethereum_node_cl != 'teku')
  ```

Comment (parithosh) [reply roles/ethereum_node/tasks/validations.yaml]: accepted suggestion :D

File: roles/blobber/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+################################################################################
+##
+## blobber container configuration
+##
+################################################################################
+blobber_container_name: blobber
+blobber_container_image: parithoshj/blobber:v0.0.1
+blobber_container_env: {}
+blobber_log_level: info
```
Comment (skylenet) [inline roles/blobber/defaults/main.yml:13]: The `blobber_log_level` var is not used anywhere. Could be removed?

File: roles/blobber/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+# Network configs
+blobber_p2p_port: 20000
+blobber_announced_ip: "{{ ansible_host }}"
```
Comment (skylenet) [inline roles/blobber/defaults/main.yml:17]: Var not used anywhere. Probably can be removed?

File: roles/blobber/defaults/main.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  - "{{ blobber_p2p_port }}:{{ blobber_p2p_port }}/udp"
+  - "127.0.0.1:{{ blobber_api_listen_port }}:{{ blobber_api_listen_port }}"
+blobber_container_networks: []
+
+# Storage configs
+# Set blobber_pubkeys to "{{ ansible_inventory_sources[0] | dirname }}/files/validator_keys/{{ inventory_hostname }}/pubkeys.json" for devnets
+blobber_validator_pubkeys: |
+  ["0xpubkey","0xpubkey2"]
+blobber_datadir: /data/blobber
+blobber_validator_key_file: /data/pubkeys.json
```
Comment (skylenet) [inline roles/blobber/defaults/main.yml:30]: Doesn't the validator key file path also need to be passed as an arg in the `blobber_container_command`?

https://github.com/marioevz/blobber/blob/main/cmd/blobber.go#L83 Says `--validator-key-file` can be used for that. One thing that confused me though, was that the argument description there mentions:

> "Path to validator key file: List of validator keys, one per line in hex format 

We currently are using a JSON array format, so which one is true?

File: roles/blobber/defaults/main.yml (hunk 3 above)
Comment (parithosh) [reply roles/blobber/defaults/main.yml:17]: fixed

**Review by skylenet — APPROVED** [review]

**Pull request merged**